src/snmalloc/backend_helpers/largebuddyrange.h

```diff
@@ -327,6 +328,17 @@ namespace snmalloc
     }
 
   public:
+    /**
+     * We use a nested Apply type to enable a Pipe operation.
+     */
+    template<typename ParentRange2>
```

Comment:
Please can we have this comment on all of the other ranges?

Reply:
I guess, I got bored about half way through.  I have a cuppa now, so can probably do the rest ;-)

Reply:
Done